pkg/agent/openflow/client.go

```diff
@@ -519,11 +556,10 @@ func (c *client) InstallEndpointFlows(protocol binding.Protocol, endpoints []pro
 		if endpoint.GetIsLocal() {
 			flows = append(flows, c.hairpinSNATFlow(endpointIP))
 		}
-		if err := c.addFlows(c.serviceFlowCache, cacheKey, flows); err != nil {
-			return err
-		}
+		keyToFlows[cacheKey] = flows
```

Comment:
IMO, `keyToFlows` is not very readable. Maybe `flowMap`?

Reply:
xToY is a widely used style to name map, I think it has more information than yMap.
And we already uses this style in many place:
https://github.com/antrea-io/antrea/blob/41ef36b87650fbe35e3b6c3e20986fc301c00e3f/pkg/agent/openflow/pipeline.go#L296
https://github.com/antrea-io/antrea/blob/41ef36b87650fbe35e3b6c3e20986fc301c00e3f/pkg/agent/nodeportlocal/k8s/npl_controller.go#L59
K8s and Golang use this style too.
https://github.com/golang/go/blob/9eee0ed4391942c73157c868a9ddcfdef48982f9/src/crypto/tls/common.go#L548
https://github.com/kubernetes/kubernetes/blob/fffaadc01331cca57cafa0fc066a2a3eec23acb8/staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go#L49

Reply:
I don't mean one style is better than another or we need to unify all names to one style. I'm fine with either style, just feel unncessary to switch to the proposed name for the above reason.

Reply:
Got it.